front/types/assistant/agent_message_content.ts

```diff
@@ -0,0 +1,42 @@
+import type { FunctionCallType } from "@app/types/assistant/generation";
+
+export type TextContentType = {
+  type: "text_content";
+  value: string;
+};
+
+export type ReasoningContentType = {
+  type: "reasoning";
+  value: {
+    displayableText?: string;
```

Comment:
reasoning?

Reply:
I want to convey that it's usually not the actual reasoning but a displayable representation of it